homeassistant/components/overkiz/coordinator.py

```diff
@@ -35,6 +35,8 @@
 class OverkizDataUpdateCoordinator(DataUpdateCoordinator[dict[str, Device]]):
     """Class to manage fetching data from Overkiz platform."""
 
+    original_update_interval: timedelta
```

Comment:
I think this can be removed since its just set as a member variable in the constructor.

Reply:
Fixed in 665a6473dc8a94581faaba41065463d36b4286e9